firmware/mcal/cli/periph/pwm.h

```diff
@@ -16,37 +17,33 @@ class PWMOutput : public shared::periph::PWMOutput {
     PWMOutput(std::string name) : name_(name) {}
 
     void Start() override {
-        std::cout << "Starting PWM " << name_ << std::endl;
+        std::cout << std::format("Starting PWN {}", name_) << std:endl; 
     }
 
     void Stop() override {
-        std::cout << "Stopping PWM " << name_ << std::endl;
+        std::cout << std::format("Stopping PWN {}", name_) << std::endl; 
     }
 
     void SetDutyCycle(float duty_cycle) override {
         duty_cycle_ =
             shared::util::Clamper<float>::Evaluate(duty_cycle, 0, 100);
 
-        std::cout << "Setting PWM " << name_ << " duty cycle to " << duty_cycle_ << "%"
-                  << std::endl;
+        std::cout << std::format("Setting PWN {} duty cycle to {:.3f} %", name_, duty_cycle_) << std::endl; 
```

Comment:
No space after percent. Good use of .3f but that'll print a lot of decimals like 93.123. Is there a way to print 3 sig figs instead? (Don't just change to .1f)